hbase-diagnostics/src/main/java/org/apache/hadoop/hbase/master/balancer/LoadBalancerPerformanceEvaluation.java

```diff
@@ -177,7 +175,7 @@ protected int doWork() throws Exception {
 
   public static void main(String[] args) throws IOException {
     LoadBalancerPerformanceEvaluation tool = new LoadBalancerPerformanceEvaluation();
-    tool.setConf(UTIL.getConfiguration());
+    tool.setConf(HBaseConfiguration.create());
```

Comment:
Effectively the same, but since this not a test, this makes more sense

Reply:
yes it should be same effectively.